### PR TITLE
fix: Using find after using findallcomponents does not match expectat…

### PR DESCRIPTION
**Pull request by llllllllllx**
Now I have fixed the problem that the use of find after using findallcomponents does not match the expectation, and the use of find after findallcomponents should be under the current node instead of the root node

### CODE REVIEW

File: src/vueWrapper.ts [hunk 1]
```diff
@@ -104,8 +104,11 @@ export class VueWrapper<T extends ComponentPublicInstance>
   ): DOMWrapper<SVGElementTagNameMap[K]>
   find<T extends Element>(selector: string): DOMWrapper<T>
   find(selector: string): DOMWrapper<Element> {
-    // force using the parentElement to allow finding the root element
-    const result = this.parentElement.querySelector(selector)
+    let result = this.parentElement['__vue_app__']
```
Comment (cexbrayat) [inline src/vueWrapper.ts:107]: nit: that should still be a const

File: src/vueWrapper.ts [hunk 2]
```diff
@@ -202,7 +205,10 @@ export class VueWrapper<T extends ComponentPublicInstance>
   ): DOMWrapper<SVGElementTagNameMap[K]>[]
   findAll<T extends Element>(selector: string): DOMWrapper<T>[]
   findAll(selector: string): DOMWrapper<Element>[] {
-    const results = this.parentElement.querySelectorAll(selector)
+    let results = this.parentElement['__vue_app__']
```
Comment (cexbrayat) [inline src/vueWrapper.ts:208]: nit: same here

File: src/vueWrapper.ts (hunk 1 above)
Comment (cexbrayat) [reply src/vueWrapper.ts:107]: Would you mind adding a comment on why we have to do this (you can even add a link to your issue)?

**Review by cexbrayat — CHANGES_REQUESTED** [review]
LGTM, thanks for the PR 👍
I just added a few comments: if you can address them, I'll merge.

File: src/vueWrapper.ts (hunk 1 above)
Comment (llllllllllx) [reply src/vueWrapper.ts]: this is indeed my mistake. I have modified and resubmitted the code

**Comment by cexbrayat** [comment]
👍 I squashed the commits and mentioned the fixed issue. Thanks for your contribution!

**Pull request merged**